pom.xml

```diff
@@ -26,7 +26,7 @@
   <properties>
     <!-- Always build file system layout for Docker images -->
     <docker.skip-build>false</docker.skip-build>
-    <!-- Never deploy to maven repository since make only builds CCloud Docker images -->
+    <!-- Never deploy to maven repository since make only builds Confluent Cloud Docker images -->
```

Comment:
Context: This will help trigger the third party notices promotion once this PR is merged.